internal/pkg/deploy/upload/asset/asset.go

```diff
@@ -41,6 +41,8 @@ func Upload(fs afero.Fs, source, destination string, opts *UploadOpts) ([]string
 		for i, f := range files {
 			paths[i] = filepath.Join(source, f.Name())
 		}
+	} else if destination == "" { // only applies to files, not folders
```

Comment:
Isn't the default destination `assets`?

Reply:
oh nvm. that's with no FileUploads.
I am making the default `/` with the selector....

Reply:
the default destination in their bucket, for just a file - will be `/<filepath>`. (that's what this is doing)

Reply:
If it's set as `/`, then destination won't be `""` though, right?

Reply:
is the `source` an absolute path (it should be right?), if so, would we be uploading to a path that looks like `/Users/admin/Projects/report-site/static/asset/meme.jpg`

Reply:
no -- it's actually just a path relative to the `copilot` directory. So given the folder `<repo>/assets/pic.jpg`, `source: assets/pic.jpg` with no destination set would upload to their s3 bucket with the key `/assets/pic.jpg`.

Reply:
"`LocalPath`: represents the path of a local file or directory. It can be written as an absolute path or relative path." https://docs.aws.amazon.com/cli/latest/reference/s3/index.html
And why would it be relative to the `copilot` dir and now the `cwd`? 
Also just a note to keep Windows users in mind as we work on this.

Reply:
i think the `cwd` doesn't really make sense for copilot - we're not picky about _where_ in a repo the `copilot` command is ran. i.e., running `cd /repo/cmd/main && copilot deploy` should deploy the same thing as `cd /repo/cmd && copilot deploy` - if we make it relative to `cwd`, the manifest source configuration will only be correct for a single location in the repository. what do you think?

Reply:
Agree!